### PR TITLE
fix annotation provider details

### PR DESCRIPTION
**Pull request by jyothishnt**
If `annotation provider version` or `last updated date` is unavailable then update payload accordingly so that it doesnt return "None" to the client

### CODE REVIEW

**Comment by azangru** [comment]
Is this a good idea? The details endpoint is very slow right now.

**Comment by jyothishnt** [comment]
> Is this a good idea? The details endpoint is very slow right now.

I dont know, lets discuss in the daily scrum

File: app/vep/vep_resources.py [hunk 1]
```diff
@@ -218,12 +218,19 @@ async def get_form_config(request: Request, genome_id: str):
     try:
         attributes = await get_genome_metadata(genome_id)
         annotation_provider_name = attributes.get("genebuild.provider_name")
-        annotation_version = attributes.get("genebuild.display_version")
-        last_updated_date = attributes.get("genebuild.last_geneset_update")
+        annotation_version = attributes.get("genebuild.display_version") or ""
```
Comment (azangru) [inline app/vep/vep_resources.py]: didn't you change the attribute name?

Comment (azangru) [reply app/vep/vep_resources.py]: Also, `.get` accepts a second argument which is the default return value

**Review by azangru — APPROVED** [review]

**Review by Mehrnaz-Charkhchi — APPROVED** [review]

**Pull request merged**